system/include/compat/sys/unistd.h

```diff
@@ -1 +1,2 @@
 #include <unistd.h>
+#include <random.h> // for getentropy()
```

Comment:
Is this file `sys/unistd.h` ever actually included by anyone?  (As opposed to `unistd.h` itself which lives elsewhere?)

Reply:
Oh, I meant to add it to `compat/unistd.h`, so yes, this was the wrong place.

Maybe it doesn't need to be added anywhere? The docs say it [should be in unistd](https://man7.org/linux/man-pages/man3/getentropy.3.html), but the usage in libc++ is for random.h. I'll remove this for now, we can add more later I guess.

Reply:
If it goes anywhere it should probably be in `./system/include/libc/unistd.h`  I guess?

Reply:
Looks like libc++ includes `#include <sys/random.h>` directly so making it part of `unistd.h` too is not necessary in this case.

Reply:
Yeah, looks like sys/random is enough.